test-app-sitebricks.yaml

```diff
@@ -51,9 +51,9 @@ paths:
           description: "invalid id of user"
```

Comment:
1. I think it would be inconvinient for the client to fill in he's own id
2. If there is something for validating you must have it in the server-side too, since the frontend is in the client machine, and someone may in verious ways send requests to you app without using the supplied frontend.